web/src/components/Upstream/constant.ts

```diff
@@ -16,49 +16,76 @@
  */
 export const DEFAULT_UPSTREAM = {
   upstream_id: '',
+  // NOTE: the following fields are the default configurations
+  // https://github.com/apache/apisix/blob/master/apisix/schema_def.lua#L325
   nodes: [
     {
       host: '',
       port: 80,
       weight: 1
     }
   ],
-  type: 'roundrobin',
+  retries: 0,
   timeout: {
+    // TODO: > 0
     connect: 6,
     send: 6,
     read: 6,
   },
-  retries: 1
+  type: 'roundrobin',
+  checks: {},
+  scheme: "http",
```

Comment:
should keep same style `""` or  `' '`.

Reply:
yes, will be formatted globally in a new PR.

Reply:
https://github.com/apache/apisix-dashboard/issues/1746